### ISSUE TITLE
Setup Nest for Angular routing

### ISSUE THREAD

**Issue by dchrzanowski** [nestjs/docs.nestjs.com#80]
Referring to an issue that I have [incorrectly placed](https://github.com/nestjs/nest/issues/694) in the framework repo.

## I'm submitting a...
<pre><code>
[ ] Regression 
[ ] Bug report
[ ] Feature request
[X] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
It is unclear which behaviour to use for serving an Angular app with routing. Should I use a `*` path controller as the last on in the array of controllers and do a `res.sendFile` to serve the `index.html` file or should I use a `HttpExceptionFilter` to `res.sendFile` the `index.html` file whenever the path is unresolved?

## What is the motivation / use case for changing the behavior?
Using the `HttpExceptionFilter` filter seems like a logical choice, because of Angular needing access to other static files. This should be clearly laid out in the documentation in my opinion.

Basically: Have a distinct pattern to serve an Angular application, since this framework is very friendly with Angular.

## Environment
NestJS 5.0.0

**Comment by pumano** [comment]
I agree with it, I have working example for it and can help with documentation. @kamilmysliwiec how do you think about it?

**Comment by kamilmysliwiec** [comment]
PR's are open @pumano 🙂Feel free to do this!

**Comment by kamilmysliwiec** [comment]
Recipe: https://docs.nestjs.com/recipes/serve-static
And for SSR: https://github.com/nestjs/ng-universal (`ng add @nestjs/ng-universal`)
